index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@400;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Zilla+Slab:ital,wght@0,300;0,400;0,600;0,700;1,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Tinos&display=swap" rel="stylesheet">
+    <title>apple.com</title>
+</head>
+<body>
+    <main class="main-box">
+        <div class="upper-section">
+            <div class="top-box">
+                <nav class="flex-fix">
+                    <ul class="top-menu flex-fix">
+                        <li><img class="logo" src="images/apple.png" alt="apple-icon"></li>
+                        <li><a href="#">Store</a></li>
+                        <li><a href="#">Mac</a></li>
+                        <li><a href="#">iPod</a></li>
+                        <li><a href="#">iPhone</a></li>
+                        <li><a href="#">iPad</a></li>
+                        <li><a href="#">iTunes</a></li>
+                        <li><a href="#">Support</a></li>
+                    </ul>
+                    <form class="nav-form" action="submit">
+                        <img src="images/search.png" alt="search">
+                        <input class="search" type="text">
+                    </form>
+                </nav>
+
+                <div class="titleMain">
+                    <div class="titleHeader"><a href="#"></a></div>
+                    <div class="titleSubHeader"><a href="#"></a></div>
+                    <div class="titleLink"><a href="#"></a></div>
```

Comment:
- [ ] I can't see the whole content of this link. Kindly fix that as well.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/88217302-be391880-cc34-11ea-968b-44acccdd03fb.png) | ![image](https://user-images.githubusercontent.com/61479648/88217329-c8f3ad80-cc34-11ea-88e6-fadce15fe6cf.png) |